src/main/java/net/dv8tion/jda/bot/sharding/DefaultShardManager.java

```diff
@@ -253,6 +254,7 @@
      *         The token
      * @param  eventManager
      *         The event manager
+     * @param eventManagerProvider 
```

Comment:
Missing space and description

Reply:
Also you should probably remove the other event manager parameter since its gone now